docs/fsharp/tutorials/getting-started/getting-started-vscode.md

```diff
@@ -1,4 +1,269 @@
 # Getting Started with F# in Visual Studio Code with Ionide
 
-> [!NOTE]
-This is still in-progress.
\ No newline at end of file
+You can write F# in [Visual Studio Code](https://code.visualstudio.com) with the [Ionide plugin](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-fsharp), to get a cross-platform, lightweight IDE experience with IntelliSense and basic code refactorings.
+
+## Prerequisites
+
+F# 4.0 or higher must be installed on your machine to use Ionide.
+
+### Windows
+
+If you're on Windows, the easiest way to do that is to install [Visual Studio with the Visual F# Tools](getting-started-visual-studio.md#installing-the-visual-f#-tools).
+
+Alternatively, you can following the following instructions:
+
+1. Install [.NET Framework 4.5 or higher](https://www.microsoft.com/en-US/download/details.aspx?id=30653) if you're running Windows 7.  If you're using Windows 8 or higher, you do not need to do this.
+
+2. Install the Windows SDK for your OS:
+
+    * [Windows 10 SDK](https://dev.windows.com/en-US/downloads/windows-10-sdk)
+    * [Windows 8.1 SDK](http://msdn.microsoft.com/windows/desktop/bg162891)
+    * [Windows 8 SDK](http://msdn.microsoft.com/windows/hardware/hh852363.aspx)
+    * [Windows 7 SDK](http://www.microsoft.com/download/details.aspx?id=8279)
+
+3. Install the [Microsoft Build Tools 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48159).
+
+4. Install the [Visual F# Tools](https://www.microsoft.com/en-us/download/details.aspx?id=48179).
```

Comment:
This is what I see:
![image](https://cloud.githubusercontent.com/assets/12971179/19173777/b2cd1694-8bde-11e6-888f-5af84f49d4a4.png)

And then this, when I click Download:
![image](https://cloud.githubusercontent.com/assets/12971179/19173795/d28aadc0-8bde-11e6-8aa9-c2a2b1d0096a.png)